templates/modules/b2b-account/quotes/view-quote.hypr.live

```diff
@@ -0,0 +1,32 @@
+﻿
+<div>
+  <h2 class="quote-name">
+    {{model.name}} {% if model.isDraft == true %} ({{labels.draft}}) {% endif %}
+    <button class="mz-button" style="margin-left: 10px;" data-mz-action="exitQuote">{{labels.exitQuote}}</button>
+    {% if model.status!=="Expired" %}
```

Comment:
This logic isn't correct. You need to hide the button if the quote status is `Completed` as no one can edit a completed quote. You also should disable (or hide) the button if the buyer or seller does not have edit permissions. I prefer disable currently.